packages/faust-nx/src/components/WordPressTemplate.tsx

```diff
@@ -2,40 +2,48 @@ import React, { PropsWithChildren } from 'react';
 import { DocumentNode, useQuery } from '@apollo/client';
 import { getTemplate } from '../getTemplate.js';
 import { SeedNode } from '../queries/seedQuery.js';
+import { WordPressTemplate } from '../getWordPressProps.js';
 import { getConfig } from '../config/index.js';
+import { usePreviewQuery } from '../hooks/usePreviewQuery.js';
 
 export type WordPressTemplateProps = PropsWithChildren<{
   __SEED_NODE__: SeedNode;
+  context?: { asPreview?: boolean };
 }>;
 
 export function WordPressTemplate(props: WordPressTemplateProps) {
+  const { __SEED_NODE__: seedNode, context } = props;
   const { templates } = getConfig();
 
   if (!templates) {
     throw new Error('Templates are required. Please add them to your config.');
   }
 
-  const { __SEED_NODE__: seedNode } = props;
   const template = getTemplate(seedNode, templates);
```

Comment:
will seedNode be resolved here without issues?

Reply:
I get this message when I try to preview the post/page
<img width="1788" alt="Screenshot 2022-08-22 at 13 00 19" src="https://user-images.githubusercontent.com/328805/185916398-e811d2ea-f7bd-49d3-b09e-08220906762d.png">

I think this happens because seedNode is `undefined`

Reply:
@theodesp Yes there is a separate ticket for executing the seed query on the client side ([MERL-439](https://wpengine.atlassian.net/browse/MERL-439))